server/server.js

```diff
@@ -148,6 +148,24 @@ module.exports = function(port, tweetSearcher, googleAuthoriser) {
         }
     });
 
+    app.put("/admin/administrators", function(req, res) {
+        try {
+            googleAuthoriser.addAdmin(req.body.email);
+            res.sendStatus(200);
+        } catch (err) {
+            res.sendStatus(404);
```

Comment:
Since the addAdmin function is using promises, the try..catch can be replaced by .then().catch(); and also for once, 404 isn't the only error that could be returned - although the only real error in this case is that there's an error writing to the admin file, so I think that'd be a 500